Mixpanel/MPNotificationViewController.m

```diff
@@ -287,7 +312,13 @@ - (void)viewWillLayoutSubviews
     parentFrame = CGRectApplyAffineTransform(parentView.frame, CGAffineTransformMakeRotation((float)angle));
 #endif
 
-    self.view.frame = CGRectMake(0.0f, parentFrame.size.height - MPNotifHeight, parentFrame.size.width, MPNotifHeight * 3.0f);
+    if (UIDeviceOrientationIsPortrait([UIDevice currentDevice].orientation) && UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPhone) {
```

Comment:
Don't think we want `UIDeviceOrientation` I think we want `UIInterfaceOrientation`, otherwise we have to begin listening for orientation changes on the device.